### PR TITLE
[feature4] New branch created and file added called file1.txt

### PR DESCRIPTION
**Pull request by AmaduWurieBah**
new branch created and new file also created in branch four

### CODE REVIEW

**Review by awbah — APPROVED** [review]
Nice work

File: feature4/file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+New file created in Feature 4 branch
```
Comment (AmaduWurieBah) [inline feature4/file1.txt:1]: nice one

**Review by AmaduWurieBah — COMMENTED** [review]
Ok good

File: feature4/file1.txt (hunk 1 above)
Comment (awbah) [reply feature4/file1.txt:1]: Ok good

**Pull request merged**